framework/test/unit/engine/consensus/certificate_generation/commit_pool.spec.ts

```diff
@@ -968,7 +1046,7 @@ describe('CommitPool', () => {
 			expect(isCommitVerified).toBeTrue();
 		});
 
-		it('should return false when aggregate commit is not signed at height maxHeightCertified', async () => {
+		it('should return false when aggregate commit is <= maxHeightCertified', async () => {
```

Comment:
1. "height" of aggregateCommit 
2. `is less than`, or remove `is` sounds better